packages/account-v2/src/containers/ProofOfIncome/ProofOfIncomeForm.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { StandaloneArrowUpFromBracketBoldIcon } from '@deriv/quill-icons';
+import { Button, Divider, Text, useDevice } from '@deriv-com/ui';
+import { FormDropDownField } from '../../components/FormFields';
+import { FormDocumentUploadField } from '../../components/FormFields/FormDocumentUploadField';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { documentValidation, listItems, PoincDocumentsList } from '../../constants/proofofincomeConstants';
+
+export const ProofOfIncomeForm = () => {
+    const { isMobile } = useDevice();
+    return (
+        <div className='relative z-0 h-full w-full  grid gap-16 '>
+            <Formik>
```

Comment:
please check the type issue reported by sonarcloud

Reply:
![Screenshot 2024-04-29 at 9 12 40 AM](https://github.com/binary-com/deriv-app/assets/158162395/77279437-103c-4943-a6c3-657a2bb5c50b)

these are the attributes that has to be included while using the Formic component but as of now there is no use of this it will be used in the other subtasks. Please let me know on what to do 🙏